front/src/modules/ui/table/states/availableTableColumnsScopedState.ts

```diff
@@ -0,0 +1,13 @@
+import { atomFamily } from 'recoil';
```

Comment:
I'm surprise we need a state for this one? it should be computed from  (static availableColumn list) - (savedTableColumnsState), isn't it?

Reply:
We need to know which list to use (People ? Companies ?). Moreover, from my understanding people/companies are related to business logic which should not be imported in the `ui` folder? Here I used the same approach as `availableFiltersScopedState`.